ui/src/app/utils/workspace-permissions.ts

```diff
@@ -37,4 +37,9 @@ export class WorkspacePermissions {
     return this.workspace.researchPurpose.reviewRequested === true &&
       this.workspace.researchPurpose.approved === false;
   }
+
+  get isReadWrite() {
+    return this.accessLevel === WorkspaceAccessLevel.WRITER &&
+        this.isOwner == false;
```

Comment:
Why this second check? I think either the method should be named differently or this part should be removed. I'm not sure what the actual desired behavior is.

Reply:
Ditto, I'm pretty sure that if the access level is Writer, then owner will have to be false. Also, if you did keep that check, `===` instead of `==`